otherlibs/unix/unix.mli

```diff
@@ -298,6 +309,8 @@ val in_channel_of_descr : file_descr -> in_channel
    [set_binary_mode_in ic false] if text mode is desired.
    Text mode is supported only if the descriptor refers to a file
    or pipe, but is not supported if it refers to a socket.
+   Under Windows, [set_binary_mode_in] always fails on channels created
```

Comment:
As non-native English speaker (but who has been speaking English daily for 11 years), I'd say "On Windows"